ci/prow/periodic_config.go

```diff
@@ -327,3 +328,71 @@ func generateClearAlertsPeriodicJob() {
 	addVolumeToJob(&data.Base, "/secrets/cloudsql/monitoringdb", "monitoring-db-credentials", true, "")
 	executeJobTemplate("periodic clearalert", readTemplate(periodicCustomJob), "presubmits", "", data.PeriodicJobName, false, data)
 }
+
+func generateIssueTrackerPeriodicJobs() {
+	staleJobName := "periodic-test-infra-stale"
+	staleLabelFilter := `
+        -label:lifecycle/frozen
+        -label:lifecycle/stale
+        -label:lifecycle/rotten`
+	staleUpdatedTime := "2160h"
+	staleComment := `|-
+        --comment=Issues go stale after 90d of inactivity.
+        Mark the issue as fresh with /remove-lifecycle stale.
+        Stale issues rot after an additional 30d of inactivity and eventually close.\n
+        If this issue is safe to close now please do so with /close.\n
+        Send feedback to Knative Productivity Slack channel or knative/test-infra.
+        /lifecycle stale`
+	generateIssueTrackerPeriodicJob(staleJobName, staleLabelFilter, staleUpdatedTime, staleComment)
+
+	rottenJobName := "periodic-test-infra-rotten"
+	rottenLabelFilter := `
+        -label:lifecycle/frozen
+        label:lifecycle/stale
+        -label:lifecycle/rotten`
+	rottenUpdatedTime := "720h"
+	rottenComment := `|-
+        --comment=Stale issues rot after 30d of inactivity.
+        Mark the issue as fresh with /remove-lifecycle rotten.
+        Rotten issues close after an additional 30d of inactivity.\n
+        If this issue is safe to close now please do so with /close.\n
+        Send feedback to Knative Productivity Slack channel or knative/test-infra.
+        /lifecycle rotten`
+	generateIssueTrackerPeriodicJob(rottenJobName, rottenLabelFilter, rottenUpdatedTime, rottenComment)
+
+	closeJobName := "periodic-test-infra-close"
+	closeLabelFilter := `
+        -label:lifecycle/frozen
+        label:lifecycle/rotten`
+	closeUpdatedTime := "720h"
+	closeComment := `|-
+        --comment=Rotten issues close after 30d of inactivity.
+        Reopen the issue with /reopen.
+        Mark the issue as fresh with /remove-lifecycle rotten.\n
+        Send feedback to Knative Productivity Slack channel or knative/test-infra.
+        /close`
+	generateIssueTrackerPeriodicJob(closeJobName, closeLabelFilter, closeUpdatedTime, closeComment)
+}
+
+func generateIssueTrackerPeriodicJob(jobName, labelFilter, updatedTime, comment string) {
+	var data periodicJobTemplateData
+	data.Base = newbaseProwJobTemplateData("knative/test-infra")
+	data.Base.Image = githubCommenterDockerImage
+	data.PeriodicJobName = jobName
+	data.CronString = issueTrackerPeriodicJobCron
+	data.Base.Command = "/app/robots/commenter/app.binary"
+
+	data.Base.Args = []string{
+		fmt.Sprintf(`|-
```

Comment:
Why not `"--query=org:knative\n" + labelFiler`?

Reply:
Done.